Templates/AppSource App/.github/workflows/UpdateGitHubGoSystemFiles.yaml

```diff
@@ -74,18 +74,44 @@ jobs:
 
       - name: Calculate Input
         env:
-          directCommit: '${{ github.event.inputs.directCommit }}'
+          directCommit: ${{ github.event.inputs.directCommit }}
           downloadLatest: ${{ github.event.inputs.downloadLatest }}
           eventName: ${{ github.event_name }}
+          GhTokenWorkflow: ${{ fromJson(steps.ReadSecrets.outputs.Secrets).ghTokenWorkflow }}
         run: |
-          $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
+          $errorActionPreference = "Stop";
+          $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
           $directCommit = $ENV:directCommit
           $downloadLatest = $ENV:downloadLatest
           Write-Host $ENV:eventName
           if ($ENV:eventName -eq 'schedule') {
-            Write-Host "Running Update AL-Go System Files on a schedule. Setting DirectCommit and DownloadLatest to true"
-            $directCommit = 'true'
             $downloadLatest = 'true'
+            $directCommit = 'true'
+            try {
+              Write-Host "Running Update AL-Go System Files on a schedule."
+              $ENV:GH_TOKEN = [System.Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String($env:GhTokenWorkflow))
```

Comment:
Do we know that the GhTokenWorkflow exists (is not empty) at this point? If not, it might be worth adding a defensive check.

Reply:
It could be empty or it could have expired - which is why I just ignore this step if it doesn't work.
The Update AL-Go System Files already provides errors if it is not good.